database/migrations/2018_11_05_164652_make_end_date_nullable_in_campaigns_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MakeEndDateNullableInCampaignsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->dateTime('end_date')->nullable()->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->dateTime('end_date')->comment('The end date of the campaign.');
```

Comment:
This "down" migration should always undo the change in the "up" migration:

```suggestion
            $table->dateTime('end_date')->nullable(false)->change();
```